tests/unit/spread-shellcheck/can-fail

```diff
@@ -2,6 +2,7 @@ tests/main/base-snaps-refresh/task.yaml
 tests/main/interfaces-calendar-service/task.yaml
 tests/main/interfaces-contacts-service/task.yaml
 tests/main/interfaces-desktop-document-portal/task.yaml
+tests/main/interfaces-joystick/task.yaml
```

Comment:
Why is it added to `can-fail`? I thought we would make sure that new task.yaml are shellcheck clean?